econet_hvac_furnace.yaml

```diff
@@ -151,12 +151,12 @@ sensor:
     filters:
       - delta: 0.9
   - platform: template
-    name: "Cooling Stage"
+    name: "Compressor Stage"
     id: cool_stage
```

Comment:
update this too?

Reply:
Not sure what exactly you're asking here?

Reply:
Done